pack/fhv/fhvc.json

```diff
@@ -118,9 +118,349 @@
+    "code": "10591",
+    "clues": 3,
+    "double_sided": true,
+    "encounter_code": "the_thing_in_the_depths",
+    "encounter_position": 4,
+    "faction_code": "mythos",
+    "flavor": "The bog is supposedly the origin of the sample that drew Dr. Marquez to the island in the first place. You have your work cut out for you.",
+    "illustrator": "Rob Laskey",
+    "name": "A Botanical Survey",
+    "pack_code": "fhvc",
+    "position": 91,
+    "quantity": 1,
+    "stage": 1,
+    "text": "<b>Objective</b> – At the end of the round, investigators at the North Shore location may spend the requisite number of clues, as a group, to advance. <i>See the Location Placement diagram on page 28 of the campaign guide for the North Short location.)</i>",
+    "type_code": "act"
+  },
+  {
+    "back_flavor": "You corner the reptilian creature in a shining pool, where it sinks into the mud, clearly exhausted. As it starts munching on a tangle of flowers, you take several sketches and notes of the thing's sprouting shell. The alkaloid blossom on its back seems to pulse. Producing a pair of shears, you prepare to take a sample.",
+    "back_name": "Strange Anatomy",
+    "back_text": "If it is <b>Day 1</b>, read the following. Otherwise, <b>(→R1)</b>.<blockquote>\"Stop!\" Dr. Marquez shouts. \"That flower is parasitic. It is now part of the creature's anatomy. There's no telling how deeply it's embedded.\" She points to the flower's thick translucent stem: vicious white fluid pumps through it like an artery. \"If the flower dies, its host dies too.\"</blockquote>\n<b>(→R2)</b> <i>(page 30)</i>",
+    "code": "10592",
+    "clues": null,
+    "double_sided": true,
+    "encounter_code": "the_thing_in_the_depths",
+    "encounter_position": 5,
+    "faction_code": "mythos",
+    "flavor": "The large flowering turtle-like creature is on the move. Perhaps you can herd it to safety.",
+    "illustrator": "Rob Laskey",
+    "name": "Discovery of a Lifetime",
+    "pack_code": "fhvc",
+    "position": 92,
+    "quantity": 1,
+    "stage": 2,
+    "text": "[fast] Investigators at a single location adjacent to Chelydran Hybrid's location spend 1[player] clues, as a group: For the remainder of the round, change Chelydran Hybrid's patrol location to that location.\n<b>Forced</b> – At the end of the enemy phase: Chelydran Hybrid takes 1 damage for each ready [[Abomination]] enemy at its location.\n<b>Objective</b> – At the end of the round, if the Chelydran Hybrid is at the Starting Location, immediately advance.",
+    "type_code": "act"
+  },
+  {
+    "back_clues": null,
+    "back_flavor": "The calm ripples hide something sinister deep below.",
+    "back_name": "Open Water",
+    "back_traits": "Sunken.",
+    "back_shroud": 5,
+    "back_illustrator": "Pixoloid Studios",
+    "back_victory": 1, 
```

Comment:
@fspoettel @zzorba Do we have such fields as 'back_shroud' and 'back_victory'? And if not, any suggestions what to do with the double-sided locations in Thing in the Depths? Maybe linked cards?

In fact, I see that the cards are numbered a and b, so I'll suggest that change.